src/sentry/integrations/source_code_management/issues.py

```diff
@@ -10,9 +10,14 @@
     SCMIntegrationInteractionType,
 )
 from sentry.integrations.source_code_management.repository import BaseRepositoryIntegration
+from sentry.integrations.utils.metrics import EventLifecycle
 from sentry.models.group import Group
 from sentry.shared_integrations.exceptions import ApiError, IntegrationError
 
+SOURCE_CODE_ISSUE_HALT_PATTERNS = {
+    "No workspace with identifier",  # Bitbucket
+}
```

Comment:
Since the SLO is also recorded here, I don't know a great way to avoid adding 'bitbucket' specific terminology to pass on these failures 🤷